tests/test_bigtable_loader.py

```diff
@@ -666,6 +668,55 @@ def test_bigtable_metadata_as_json_execution_order(
     }
 
 
+def test_table_creation(instance_id: str, client: bigtable.Client) -> None:
+    table_id = TABLE_ID_PREFIX + "".join(
+        random.choice(string.ascii_lowercase) for _ in range(10)
+    )
+    # Create table.
+    init_document_table(instance_id, table_id, client)
+
+    # Assert table exists.
+    table_client = client.instance(instance_id).table(table_id)
+    assert table_client.exists()
+    assert sorted(table_client.list_column_families().keys()) == ["langchain"]
+    # Expect second creation to fail.
+    with pytest.raises(AlreadyExists):
+        init_document_table(instance_id, table_id, client)
+
+    # Delete table.
+    table_client.delete()
+    assert not table_client.exists()
+
+    # Create with column families.
+    content_column_family = "content_column_family"
+    first_column_from_mapping = "first_column_from_mapping"
+    second_column_from_mapping = "second_column_from_mapping"
+    metadata_as_json_column_family = "metadata_as_json_column_family"
+    init_document_table(
+        instance_id,
+        table_id,
+        client,
+        content_column_family,
+        [
+            MetadataMapping(first_column_from_mapping, "", "", Encoding.ASCII),
+            MetadataMapping(second_column_from_mapping, "", "", Encoding.ASCII),
+        ],
+        metadata_as_json_column_family,
+    )
+    expected_families = [
+        content_column_family,
+        first_column_from_mapping,
+        second_column_from_mapping,
+        metadata_as_json_column_family,
+    ]
+
+    # Assert successful creation.
+    assert table_client.exists()
```

Comment:
Do we need to delete the table?

Reply:
We do D=
Used the table_id parameter to take care of it, regardless of test failures